### PR TITLE
[DOC] Clarify communication pathways

### LINKED ISSUE #478: Clarify communication methods for tedana
### Summary
Currently, there are three main avenues by which we communication about tedana: 1) on the Github through issues and pull request reviews, 2) on the Gitter chat room, and 3) through the newsletter.

Thus far, I would suggest that conversation seems to move from the Gitter chat room (2), to the Github repository (1), and -- if efforts are successful in implementing some new feature or patching a bug -- they are advertised in (3) the newsletter. I don't think this is a desirable routing of information, and I'd like to strongly suggest we move to a routing that looks like (1) --> (3). This isn't to say that the Gitter chat room can't exist, but I think it shouldn't be a source of information in and of itself.

Instead, we should be primarily communicating on the Github repository, with the Gitter chat room for redirecting to existing (open or closed) Github issues / PRs. This will be especially useful as we start expanding into multiple repositories, if someone is having difficulty locating the appropriate issue. 

Previously our informal rule was that "quick" questions should go on the Gitter, but it seems that this is a subjective line and as such we've often seen extended conversations on Gitter. The problem with this is that Gitter is hard to index, and therefore information is likely to be (and has been repeatedly) lost over time.

### Additional Detail

In some of her Turing Way work, @KirstieJane has  made a great point about open source projects in general -- they're often hard to adjust to for researchers used to "conversations in the kitchen;" that is, the idea that you can quickly stop over at your colleague's desk and have a discussion on some shared research project. 

These conversations are obviously helpful for many teams working in traditional environments, but in an open source context they're actually detrimental because the information is not recorded and cannot be distributed more broadly. You therefore create working conditions where only some members of the team have access to information, making it more difficult for new contributors to onboard and for contributors in disparate time zones to feel empowered to address issues or open PRs.

I think we've fallen into a comfort zone of using Gitter for our "conversations in the kitchen." Although we could close down Gitter, I don't think this is the right approach. Instead, I'd like us to all consciously make an effort to minimize our conversations on Gitter and instead take advantage of the GitHub issue system. 

### Next Steps

I'd love to get input from @tsalo @smoia @CesarCaballeroGaudes @eurunuela @javiergcas @handwerkerd @jbteves @dowdlelt. 

We should also make sure the relevant contributing guidelines are updated accordingly.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #478.

Changes proposed in this pull request:

- Add section in Support and Communication page on how and when each of the established communication pathways should be used.

### CODE REVIEW

**Comment by jbteves** [comment]
This looks good. Do we want to include a link to our [newsletter repo](https://github.com/ME-ICA/newsletters), however?

File: docs/support.rst [hunk 1]
```diff
@@ -15,3 +17,17 @@ We will also attempt to archive certain common questions and associate answers i
...
 .. _tedana tag: https://neurostars.org/tags/tedana
+
+Internal Communication Pathways
+===============================
+
+There are multiple mechanisms for development-related communication, which can lead to confusion.
+Here we attempt to clarify when and how each communication pathway should be used:
+
+- GitHub Issues and Pull Requests: Issues and PRs should be the **main avenue of communication** for questions and proposals related to tedana's development.
+- Gitter: While tedana does have a Gitter channel, this channel should not be used discussions. Instead, its primary use is to redirect to **existing** Issues and PRs, whether open or closed.
```
Comment (eurunuela) [inline docs/support.rst:28]:
```suggestion
- Gitter: While tedana does have a Gitter channel, this channel should not be used for discussions. Instead, its primary use is to redirect to **existing** Issues and PRs, whether open or closed.
```

**Review by eurunuela — APPROVED** [review]
LGTM! Thanks Taylor!

File: docs/support.rst [hunk 2]
```diff
@@ -15,3 +17,17 @@ We will also attempt to archive certain common questions and associate answers i
...
+- Monthly Newsletter: The monthly newsletter is used to make announcements from the tedana developers to the tedana users. Discussions that start in Issues or PRs may move to the newsletter if they are news-worthy.
```
Comment (tsalo) [inline docs/support.rst:29]:
```suggestion
- Monthly Newsletter: The `monthly newsletter`_ is used to make announcements from the tedana developers to the tedana users. Discussions that start in Issues or PRs may move to the newsletter if they are news-worthy.
```

File: docs/support.rst [hunk 3]
```diff
@@ -15,3 +17,17 @@ We will also attempt to archive certain common questions and associate answers i
...
+- NeuroStars: The NeuroStars forum is to be used by tedana users to ask questions about how to use tedana. tedana's developers may answer usage questions on NeuroStars, but development-related discussions should happen in GitHub Issues and PRs.
+- Mattermost: There is a `tedana channel in the BrainHack Mattermost`_. This is primarily used for online discussions during BrainHack-related hackathons (for example, the annual OHBM hackathon), if and when there is a tedana-related project. **This channel is not regularly monitored by tedana developers** and should not be used outside of hackathons.
+
+.. _tedana channel in the BrainHack Mattermost: https://mattermost.brainhack.org/brainhack/channels/tedana
```
Comment (tsalo) [inline docs/support.rst:33]:
```suggestion
.. _monthly newsletter: https://github.com/ME-ICA/newsletters
.. _tedana channel in the BrainHack Mattermost: https://mattermost.brainhack.org/brainhack/channels/tedana
```

**Comment by tsalo** [comment]
Cancelling the integration tests because this only impacts the documentation. I'll use my admin privileges to merge once I have the necessary reviews.

**Review by eurunuela — APPROVED** [review]

**Review by jbteves — APPROVED** [review]
LGTM, thanks @tsalo !

**Pull request merged**